Assets/Scripts/Drone/PictogramBehavior.cs

```diff
@@ -56,6 +56,14 @@ public void ChangePictogramImage(object data)
             throw new System.ArgumentException("data must be Sprite: " + data);
         }
     }
+    public void ChangePictogramImage(Sprite data)
```

Comment:
I think you can just use the override which takes in an object, to reduce repeated code.